enterprise/frontend/src/metabase-enterprise/metabot/components/MetabotChat/MetabotChat.tsx

```diff
@@ -248,42 +261,70 @@ export const MetabotChat = () => {
 };
 
 const Message = ({
```

Comment:
Admittedly this component is getting a bit gnarly. Happy to refactor it in another PR. I need design feedback as every UI change I've made as part of streaming was me doing my own thing since Maz was OOO.